go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	cosmossdk.io/errors v1.0.0
 	cosmossdk.io/log v1.2.0
 	cosmossdk.io/math v1.0.1
-	cosmossdk.io/store v1.0.0-alpha.1.0.20230728080422-54ed7dab3982
+	cosmossdk.io/store v1.0.0-alpha.1.0.20230807110739-60198f077af7
```

Comment:
why the bump to store?

Reply:
https://github.com/cosmos/cosmos-sdk/pull/17294